client/src/modules/StudentDashboard/components/AvailableReviewCard/AvailableReviewCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { Divider, Progress, Row, Typography } from 'antd';
+import { AvailableReviewStatsDto } from 'api';
+import CommonCard from '../CommonDashboardCard';
+
+type Props = {
+  availableReviews: AvailableReviewStatsDto[];
+  courseAlias: string;
+};
+const { Text, Link } = Typography;
+
+export function AvailableReviewCard({ availableReviews, courseAlias }: Props) {
+  return (
+    <CommonCard
+      title="Cross-check [Review]"
+      content={
+        <>
+          {availableReviews.length ? (
+            <>
+              {availableReviews.map((el, index) => (
+                <Row key={el.id} gutter={[0, 6]}>
```

Comment:
As I remember, all offsets should be divided by 4
```suggestion
                <Row key={el.id} gutter={[0, 8]}>
```

Reply:
fixed